spec.bs

```diff
@@ -4070,7 +4070,7 @@ dictionary StorageInterestGroup : AuctionAdInterestGroup {
 *This first introductory paragraph is non-normative.*
 
 {{Window/navigator}}.{{Navigator/createAuctionNonce()}} creates an <dfn>auction nonce</dfn>, a
-one-time [=version 4 UUID=] uniquely associated with a single call to
+one-time [=version 4 UUID=] [=string=] uniquely associated with a single call to

```

Comment:
Is there a way to say "the canonical string representation of a [=version 4 UUID=]", and if so, can we use that here?

Reply:
Done -- I also clean up this whole paragraph a bit, PTAL.

Reply:
Does this new wording look OK?

Reply:
Sorry, I didn't realize you were blocked on this comment. The wording LGTM modulo one small comment. Thanks!

Reply:
No worries, and thanks :)